terraform/environments/core-network-services/firewall-rules/development_rules.json

```diff
@@ -348,5 +348,19 @@
     "destination_ip": "${noms-mgmt-vnet}",
     "destination_port": "138",
     "protocol": "UDP"
+  },
+  "DOM1_atos_arkc_ras_vpn_to_mp_development": {
+    "action": "PASS",
+    "source_ip": "10.175.0.0/16", 
```

Comment:
We should use the variable provided and not hardcode the range.

Reply:
Fixed